src/features/theme/themes/windglass/styles/index.css

```
@@ -0,0 +1,811 @@
+    .wg-shell {
+      padding-inline: 0;
+      padding-block-start: 4rem;
+    }
+  }
+
+  .wg-kicker {
+    font-size: 0.6875rem;
+    font-weight: 650;
+    text-transform: uppercase;
+    letter-spacing: 0.16em;
+    color: color-mix(in srgb, currentColor 42%, transparent);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)